aesara/link/numba/dispatch/elemwise.py

```diff
@@ -424,8 +425,19 @@ def axis_apply_fn(x):
 
 @numba_funcify.register(Elemwise)
 def numba_funcify_Elemwise(op, node, **kwargs):
+    # A FunctionGraph isn't a Node in the type hierarchy, but
```

Comment:
I'm not really sure what to do here.
Previously, we just passed the node of the surrounding op, but I think this clearly isn't right, and leads to errors if we want to generate a object mode fallback for parts of the elemwise code.

So I thought we can just create a new node using `op.scalar_op.make_node(*scalar_inputs)`, but this doesn't always work (maybe actually due to a bug). Elemwise will sometimes throw errors because it thinks something in the graph isn't a scalar op.

So my conclusion was that we can just pass in the fgraph, which also has input and output arguments just as a node, and I think pretty much functions as the node conceptually. But this isn't exactly clean.

Maybe we need to figure out what's going wrong with the make_node call?

Reply:
^ @brandonwillard if you have an idea about what to do here, that would be great.

Reply:
> I'm not really sure what to do here.
> Previously, we just passed the node of the surrounding op, but I think this clearly isn't right, and leads to errors if we want to generate a object mode fallback for parts of the elemwise code.

Are you trying to make the scalar `Op` call in `numba_funcify_Elemwise` return an object-mode fallback implementation?